python/lsst/obs/ctio0m9/ingest.py

```diff
@@ -107,7 +168,15 @@ def _translateFromCalibId(self, field, md):
         return match.groups()[0]
 
     def translate_filter(self, md):
+        print('\nMerlin says there might be a problem here\nCheck what is happening when you see this\n')
```

Comment:
This should be a {{log.warn}} message.  And maybe include the commented information below.